snippets/core/system-text-json/csharp/ImmutablePoint.cs

```diff
@@ -3,17 +3,14 @@
     // <SnippetImmutablePoint>
     public struct ImmutablePoint
```

Comment:
I'd consider making this a readonly struct:

```suggestion
    public readonly struct ImmutablePoint
```